Packs/mnemonicMDR/Integrations/ArgusManagedDefence/ArgusManagedDefence.py

```diff
@@ -459,7 +461,14 @@ def download_attachment_command(args: Dict[str, Any]) -> Any:
     if not attachment_id:
         raise ValueError("attachment id not specified")
 
-    result = download_attachment(caseID=case_id, attachmentID=attachment_id)
+    try:
+        result = download_attachment(caseID=case_id, attachmentID=attachment_id)
+    except ArgusException as e:
+        return_warning(
+            message=str(e),
+            warning=e.parsed_resp if e.parsed_resp else None,
+            exit=True
```

Comment:
This is still going to exit the script. so you can use `return_error` instead.

Reply:
Then I revert to how it was before, since the Exception that the API raises is caught in main(). I want to return an error when someone is trying to download a file that doesn't exit / doesn't have access rights to.